src/wallet/wallet.cpp

```diff
@@ -281,37 +280,48 @@ CPubKey CWallet::GenerateNewKey()
             BIP44CoinType(),
             ZCASH_LEGACY_ACCOUNT).value();
 
-    std::optional<std::pair<CExtKey, HDKeyPath>> extKey = std::nullopt;
+    std::optional<std::pair<CKey, HDKeyPath>> extKey = std::nullopt;
```

Comment:
The variable is still called `extKey` even though it no longer is. (I'm assuming that `ext` stood for `extended` rather than `external`.)